vital/tests/test_image.cxx

```diff
@@ -36,6 +37,33 @@ value_at( unsigned int i, unsigned int j, unsigned int k )
   return static_cast<byte>( ( ( ks * k ) + ( js * j ) + ( is * i ) ) % 255 );
 }
 
+// ----------------------------------------------------------------------------
+void check_equal( image_memory mem_1, image_memory mem_2 )
+{
+  char* mem_1_data = (char*) mem_1.data();
+  char* mem_2_data = (char*) mem_2.data();
+
+  EXPECT_EQ( mem_1.size(), mem_2.size() );
+  for(unsigned int i=0; mem_1_data[i]!='\0'; i++)
+  {
+    EXPECT_EQ( mem_1_data[i], mem_2_data[i] );
+  }
+}
+
+// ----------------------------------------------------------------------------
+void check_equal( image img_1, image img_2 )
```

Comment:
This should be 
```c++
template <typename T>
void check_equal( image_of<T> const& img_1, image const& img_2 )
```
to match what it is replacing below.  It is testing that these accessors for `image_of<T>` and `image` return the same thing after converting one to the other.